### ISSUE TITLE
Handle Transactions with multiple operations

### ISSUE THREAD

**Issue by JamesSmartCell** [AlphaWallet/alpha-wallet-android#1304]
So far we only handle one transaction result decoded from the input. Some transactions have multiple operation results eg tx Hash ```0x6816d68d89d30129bf66fe93c03839a16955c7f4042deee09a249532bb109f9d``` on main net

[https://api.etherscan.io/api?module=account&action=txlistinternal&txhash=0x6816d68d89d30129bf66fe93c03839a16955c7f4042deee09a249532bb109f9d](https://api.etherscan.io/api?module=account&action=txlistinternal&txhash=0x6816d68d89d30129bf66fe93c03839a16955c7f4042deee09a249532bb109f9d)

Will need a suitable UI maybe with a TX tree graph.

**Comment by colourful-land** [comment]
In this case, is transaction `0x6816` one that calls contract 
`0xa57bd00134b2850b2a1c55860c9e9ea100fdd6cf` which internally did 2 Ether transfers?

**Comment by JamesSmartCell** [comment]
Yes, see: https://etherscan.io/tx/0x6816d68d89d30129bf66fe93c03839a16955c7f4042deee09a249532bb109f9d

**Comment by JamesSmartCell** [comment]
We should at least summarise all the outputs if there are side-effects not apparent from the transaction analysis. I will self assign this, but leave producing a transaction graph.
